lib/doorkeeper/orm/active_record.rb

```diff
@@ -25,11 +25,11 @@ def self.check_requirements!(_config)
              Doorkeeper::Application.table_name
            )
           unless Doorkeeper::Application.new.attributes.include?("scopes")
-            fail <<-MSG.squish
+            puts <<-MSG.squish
```

Comment:
If you fail here, the migration that adds `scopes` to `oauth_applications` will just be aborted.